test/operators.jl

```diff
@@ -89,3 +89,9 @@ end
     @test filter(!isupper, str) == replace(str, r"[A-Z]", "")
     @test filter(!islower, str) == replace(str, r"[a-z]", "")
 end
+
+# issue #19891
+B=0 .< [1 -1 5] .< 3
+@test ifelse(B, 1, 2) == [1 2 2]
```

Comment:
didn't realize ifelse is vectorized like this, probably just test the value and type of B